gnat2goto/driver/tree_walk.adb

```diff
@@ -2343,63 +2343,219 @@ package body Tree_Walk is
+              Name_Interrupt_Handler |
+            --  If the pragma appears in a protected_definition, then the
+            --  corresponding procedure can be attached ynamically, as a
+            --  handler, to interrupts. -> Ignored
               Name_Controlled |
-              Name_Convention |
-              Name_Discard_Names |
-              Name_Elaborate |
-              Name_Elaborate_All |
-              Name_Elaborate_Body |
+            --  Used to prevent any automatic reclamation of storage (garbage
+            --  collection) for the objects created by allocators of a given
+            --  access type. -> Ignored
               Name_Export |
-              Name_Import |
-              Name_Inline |
-              Name_Inspection_Point |
-              Name_Interrupt_Handler |
+            --  Used to export an Ada entity to a foreign language, thus
+            --  allowing an Ada subprogram to be called from a foreign
+            --  language, or an Ada object to be accessed from a foreign
+            --  language. -> Ignored
               Name_Linker_Options |
+            --  Used to specify the system linker parameters needed when a
+            --  given compilation unit is included in a partition. -> Ignored
+              Name_Discard_Names |
+            --  Used to request a reduction in storage used for the names of
+            --  certain entities. -> Ignored
+            Name_Inline |
+            --  Indicates that inline expansion is desired for all calls to
+            --  that entity. -> Ignored
+            Name_Inspection_Point |
+            --  Identifies a set of objects each of whose values is to be
+            --  available at the point(s) during program execution
+            --  corresponding to the position of the pragma in the compilation
+            --  unit. -> Ignored
               Name_List |
-              Name_Locking_Policy |
-              Name_Normalize_Scalars |
+            --  Takes one of the identifiers On or Off as the single
+            --  argument. It specifies that listing of the compilation is to be
+            --  continued or suspended until a List pragma with the opposite
+            --  argument is given within the same compilation. -> Ignored
+              Name_Page |
+            --  Specifies that the program text which follows the pragma should
+            --  start on a new page (if the compiler is currently producing a
+            --  listing). -> Ignored
               Name_Optimize |
+            --  Gives advice to the implementation as to whether time or space
+            --  is the primary optimization criterion. -> Ignored
               Name_Pack |
-              Name_Page |
-              Name_Preelaborate |
+            --  Specifies that storage minimization should be the main
+            --  criterion when selecting the representation of a composite
+            --  type. -> Ignored
```

Comment:
"Known but unsupported".  The reason is that things like packing fields in structures are supported by CPROVER (compare `struct s { char a; char b; int x}` and `struct t { char a; int x; char b; }`.  I'm hoping to not need to implement Ada packing but I'd like a warning if someone is using it.

Reply:
Ok.